client-app/shared/payment/components/bank-card-form.vue

```diff
@@ -117,38 +123,50 @@ const labels = computed(() => {
   };
 });
 
-const monthYupSchema = string()
+const monthYupSchema = yup
+  .string()
   .required()
   .length(2)
   .matches(/^(0?[1-9]|1[0-2])$/, t("shared.payment.authorize_net.errors.month"))
   .label(labels.value.monthLabel);
 
 const validationSchema = toTypedSchema(
-  object({
-    number: string().required().min(12).max(19).label(labels.value.number),
-    cardholderName: string().required().max(64).label(labels.value.cardholderName),
+  yup.object({
+    number: yup.string().required().min(12).max(19).label(labels.value.number),
```

Comment:
CARD_NUMBER_LENGTH_MIN
CARD_NUMBER_LENGTH_MAX
NAME_LENGTH_MAX
SECURITY_CODE_MIN
SECURITY_CODE_MAX

Reply:
Why? It usually needed to one of two:
1. To reuse (there is no such a thing)
2. To avoid magic numbers (schema is already readable)
And we also never did it and it's ok, why start here?

Reply:
MAX_PASS_LENGTH